mycroft/client/speech/hotword_factory.py

```diff
@@ -359,8 +384,23 @@ def found_wake_word(self, frame_data):
 class PorcupineHotWord(HotWordEngine):
     def __init__(self, key_phrase="hey mycroft", config=None, lang="en-us"):
         super(PorcupineHotWord, self).__init__(key_phrase, config, lang)
+
+        xdg_data_path = None
+
+        # Use the old path if available
+        old_path = join(expanduser('~'), '.mycroft', 'Porcupine')
+        if os.path.isdir(old_path):
+            xdg_data_path = old_path
```

Comment:
can we `shutil.move` the file to the xdg directory automatically instead?

Reply:
Also user installed so I don't think it should be moved. But will not be an issue when #2722 or the porcupine plugin is used.